Source: http://bugs.python.org/issue8193
Upstream: Yes
Reason: Fix test_zlib failure with zlib 1.2.4.

From d52a23fa767b9df42df5adec8ea97723814f995b Mon Sep 17 00:00:00 2001
From: antoine.pitrou <antoine.pitrou@6015fed2-1504-0410-9fe1-9d1591cc4771>
Date: Tue, 6 Apr 2010 17:24:02 +0000
Subject: [PATCH] Merged revisions 79848 via svnmerge from
 svn+ssh://pythondev@svn.python.org/python/trunk

........
  r79848 | antoine.pitrou | 2010-04-06 19:21:09 +0200 (mar., 06 avril 2010) | 3 lines

  Issue #8193: Fix test_zlib failure with zlib 1.2.4.
........


git-svn-id: http://svn.python.org/projects/python/branches/release26-maint@79850 6015fed2-1504-0410-9fe1-9d1591cc4771
---
 Lib/test/test_zlib.py |    2 +-
 Misc/NEWS             |    5 +++++
 2 files changed, 6 insertions(+), 1 deletions(-)

diff --git a/Lib/test/test_zlib.py b/Lib/test/test_zlib.py
index adb87ff..d1e1796 100644
--- a/Lib/test/test_zlib.py
+++ b/Lib/test/test_zlib.py
@@ -81,7 +81,7 @@ class ExceptionTestCase(unittest.TestCase):
 
     def test_baddecompressobj(self):
         # verify failure on building decompress object with bad params
-        self.assertRaises(ValueError, zlib.decompressobj, 0)
+        self.assertRaises(ValueError, zlib.decompressobj, -1)
 
     def test_decompressobj_badflush(self):
         # verify failure on calling decompressobj.flush with bad params
diff --git a/Misc/NEWS b/Misc/NEWS
index 92bef1c..3595306 100644
--- a/Misc/NEWS
+++ b/Misc/NEWS
@@ -72,6 +72,11 @@ Build
   variable anymore.  It also forwards the LDFLAGS settings to the linker
   when building a shared library.
 
+Tests
+-----
+
+- Issue #8193: Fix test_zlib failure with zlib 1.2.4.
+
 
 What's New in Python 2.6.5?
 ===========================
-- 
1.7.0.4

